src/main/java/com/CezaryZal/entity/health/calendar/UserAuthentication.java

```diff
@@ -0,0 +1,25 @@
+package com.CezaryZal.entity.health.calendar;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "users_hc")
+@Data
+@NoArgsConstructor
+public class UserAuthentication {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String loginName;
+    private String password;
+    private boolean approved;
+    private String roles;
```

Comment:
To samo co wyżej

Reply:
Encja połączona z bazą danych, pozwalająca uwierzytelnić wpisywane dane użytkownika oraz stworzyć token